guide/how-it-works/ecash/design-best-practices.md

```diff
@@ -0,0 +1,573 @@
+
+{% include tip/open.html color="blue" icon="info" label="Single Use Recovery" %}
+
+After successfully restoring your wallet using a recovery seed phrase, it is highly recommended that you generate a new recovery seed phrase immediately. This step is crucial because the original recovery phrase has now been used and could be more susceptible to synchronization risks.
+
+{% include tip/close.html %}
+
+#### Backup Process
+{% include image-gallery.html pages = page.images_backup %}
+
+1. **Deterministic Wallet with Seed Phrase**: Cashu uses a deterministic wallet model, where all cryptographic keys and bitcoin-backed ecash tokens can be derived from a single seed phrase. This seed phrase can be generated when the wallet is first created, or a at any point in time when the user wants to backup their wallet.
+
+2. **Secure Storage**: Users are advised to store their seed phrase securely. It is crucial to keep this seed phrase in a safe, physical format (such as written on paper) and stored in a secure location to prevent unauthorized access or loss. The best practices in the [Backup & recovery]({{ '/guide/daily-spending-wallet/backup-and-recovery/landing-page/' | relative_url }}) section also apply to Cashu wallets. 
+
+{% include tip/open.html color="blue" icon="info" label="Mint Information Needed" %}
+
+Keep a record of the mints you are connected to. During the restoration process, you will need to reconnect to these specific mints to re-verify your tokens. Without this information, you will not be able to fully restore your wallet.
+
+{% include tip/close.html %}
+
+#### Restoration Process
+{% include image-gallery.html pages = page.images_restore %}
+
+1. **Seed Phrase Entry**: When restoring a wallet, users enter their seed phrase into their wallet. This seed phrase regenerates secrets that may have been used to mint ecash.
+
+2. **Mint(s) Entry**: Users must input the mint URL for each mint they used before. This ensures their wallet can properly reconnect and recreate their tokens.
+
+3. **Verification**: The mint(s) checks these blinded secrets against its records to confirm if it has been seen before and can reissue the signature.
+
+{% include tip/open.html color="blue" icon="info" label="Privacy Considerations During Verification" %}
+
+Verifying tokens after restoration might temporarily compromise their privacy. The mint needs to revalidate the tokens, which can potentially expose the transaction history associated with those tokens. Be mindful of this aspect if privacy is a significant concern for your use case.
+
+{% include tip/close.html %}
+
+### Metadata and Information Management
+
+#### Cashu Metadata Fields: Explanation and Usage
```

Comment:
I get where you're coming from. The only times the screen really changes are for MOTD and supported features, so it could probably be compressed significantly. I'll leave it as is for now, but during the new PR review session, I'll make sure to bring it up and get everyone's thoughts on it.